website/docs/cloud-docs/projects/best-practices.mdx

```diff
@@ -0,0 +1,36 @@
+---
+page_title: Best Practices - Projects - HCP Terraform
+description: >-
+  Best practices to structure your configuration and HCP Terraform
+  projects
+---
+
+# Project Best Practices
+
+Projects let you group and scope access to your workspaces. You can group related  workspaces into projects and give teams more permissive access to individual projects rather than granting them permissions to the entire organization. 
+
+Projects offer several advantages to help you further develop your workspace strategy:
+
+- **Increased workspace organization**: You can add related workspaces to projects to simplify and organize a team's workspace view.
+- **Simplified management**: You can create project-level permissions and variable sets that apply to all current and future workspaces in the project. For example, you can create a project variable set containing your cloud provider credentials and all workspaces in the project will have access to them.
+- **Reduced risk with centralized control**: You can scope project permissions to only grant teams administrator access to the projects and workspaces they need.
+- **Efficiency through self-service**: Granting users project-level administrator permissions lets them create and manage workspaces.
+
+When using projects, we recommend the following:
+
+- Automate the creation of projects, variable sets, and teams together using the [TFE provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs).
+- Restrict the number of project administrators similar to maintain least privilege.
+- Designate a “landing zone” project that will contain workspaces used to create all other projects, teams, and workspaces. This lets you have a variable set that includes the organization token, which the TFE provider can use to create other resources in your organization. You can also create a [Sentinel policy](/terraform/cloud-docs/policy-enforcement) to prevent users in other projects from accessing the organization token.
```

Comment:
maybe for consistency, this list should also have a bolded key idea leading each bullet

Reply:
id also swap the order of 2 and 3, since bullets 1 and 2 are related